app/models/order.rb

```diff
@@ -1,5 +1,20 @@
 class Order < ApplicationRecord
   has_many :order_items
+  validates :name, :email, :mailing_address, :cc_name, :cc_cvv, :cc_exp, :cc_number, :zipcode, :status, presence: true
+  validates :email, format: { with: /\w+@\w+\.\w+/, message: "Please enter valid email address" }
+  validates :cc_cvv, :cc_number, :zipcode, numericality: { only_integer: true }
+  validates :cc_number, length: {is: 15}
+  validates :cc_cvv, length: {is: 3}
+  validates :zipcode, length: {is: 5}
 
+  def order_total
+    total = 0
+    self.order_items.each do |order_item|
+      price = order_item.product.price
+      total += price
+    end
+
+    return (total * 1.1)
+  end
 
```

Comment:
It seems like "quantity" is missing in this calculation. Please note that I made a #subtotal instance method in the OrderItem model, which we should be able to incorporate into this method. We will need to merge both PR's first👍

Reply:
i've made the change